src/System.Text.Json/src/System/Text/Json/Node/JsonBoolean.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Text.Json
+{
+    /// <summary>
+    ///   Represents a boolean JSON value.
+    /// </summary>
+    public class JsonBoolean : JsonNode, IEquatable<JsonBoolean>
```

Comment:
Seems we should seal these types?  Presumably JsonNode's internal ctor is to prevent types deriving from it, but the public ctors here would allow these types to be derived from.